src/sugestao/SugestaoService.ts

```
@@ -0,0 +1,28 @@
+import {
+  SugestaoPayload,
+  Curso,
+  Servidor,
+} from '@/commons/interfaces'
+
+class SugestaoService extends BaseService {
+  constructor() {
+    super('') // O BaseService lida com o prefixo da API_URL
+  }
+
+  async criarSugestao(sugestaoData: SugestaoPayload): Promise<any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)